cups/translate-attrs.go

```diff
@@ -293,6 +308,21 @@ func convertMarkers(printerTags map[string][]string) (*[]cdd.Marker, *cdd.Marker
 	return &markers, &states
 }
 
+//fixMarkers corrects some drivers' marker names/types where CUPS detects names/types with a comma
+// as two separate values. The second value of these pairs contain a space, so it's easy to detect.
+func fixMarkers(values []string) []string {
+	var newValues []string
+
+	for i := range values {
+		if i > 0 && len(values[i]) > 1 && values[i][0] == ' ' {
+			newValues[len(newValues)-1] = strings.Join([]string{newValues[len(newValues)-1], values[i]}, ",")
```

Comment:
I used strings.Join originally, and I assumed during this review that fmt.Sprintf() would be faster. Good to know.

I'm not that worried about it.